platform/ios/CHANGELOG.md

```diff
@@ -8,10 +8,14 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 * Deprecated `+[NSExpression featurePropertiesVariableExpression]` use `+[NSExpression featureAttributesVariableExpression]` instead. ([#11748](https://github.com/mapbox/mapbox-gl-native/pull/11748))
 
+
+### Annotations
+* Fixed an issue where selecting an onscreen annotation could move the map unintentionally. ([#11731](https://github.com/mapbox/mapbox-gl-native/pull/11731))
+* Fixed an issue where annotation views could become distorted if `rotatesToMatchCamera` is `YES`. ([#11756](https://github.com/mapbox/mapbox-gl-native/issues/11756))
```

Comment:
Shouldn't this blurb refer to `([#11817](https://github.com/mapbox/mapbox-gl-native/pull/11817))` instead?

Reply:
Good question - do we normally refer to the source ticket or the PR in the change log?

Reply:
We always refer to the PR that fixed the issue.